app/models/rules_of_origin/steps/subdivision.rb

```diff
@@ -0,0 +1,30 @@
+module RulesOfOrigin
+  module Steps
+    class Subdivision < Base
+      self.section = 'originating'
+
+      delegate :description, to: :commodity, prefix: true
+
+      attribute :subdivision
+      validates :subdivision, inclusion: { in: :available_subdivisions }
+
+      def skipped?
+        @store['sufficient_processing'] == 'no' || options.none?
```

Comment:
should we check for the capitalized `NO` ?
(maybe downcasing it?)

Reply:
This is a value from a radio button which we control so I think its fine to rely on our own Ids